scripts/run_driver_tests.psm1

```diff
@@ -267,4 +267,41 @@ function Invoke-CICDStressTests
     popd
 }
 
+function Invoke-CICDPerformanceTests
+{
+    param(
+        [parameter(Mandatory = $true)][bool] $VerboseLogs,
+        [parameter(Mandatory = $true)][bool] $CaptureProfile)
+    Push-Location $WorkingDirectory
+
+    Write-Log "Executing eBPF kernel mode performance tests."
+
+    $LASTEXITCODE = 0
+
+    # Stop the services, remove the driver from verifier, and restart the services.
+    net.exe stop ebpfsvc
+    net.exe stop ebpfcore
+    verifier.exe /volatile 0
```

Comment:
What does this command do   verifier.exe usage syntax only shows /volatile syntax as:
```
    verifier /volatile /flags <options> [<options> ...]
    verifier /volatile /adddriver <name> [<name> ...]
    verifier /volatile /removedriver <name> [<name> ...]
    verifier /volatile /faults [probability [pool_tags [applications [delay_minutes]]]]
```
None of the above match this line.

Reply:
Added more comments.
```
C:\Windows\system32>verifier.exe /volatile

Verifier Volatile Flags: 0x00120000

  Volatile Flags:

    [ ] 0x00000001 Special pool.
    [ ] 0x00000002 Force IRQL checking.
    [ ] 0x00000004 Randomized low resources simulation.
    [ ] 0x00000008 Pool tracking.
    [ ] 0x00000010 I/O verification.
    [ ] 0x00000020 Deadlock detection.
    [ ] 0x00000080 DMA checking.
    [ ] 0x00000200 Force pending I/O requests.
    [ ] 0x00000400 IRP logging.
    [ ] 0x00000800 Miscellaneous checks.

The new settings are in effect until the system
is restarted or the settings are changed again.

C:\Windows\system32>verifier.exe /volatile 0

Verifier Volatile Flags: 0x00000000

  Volatile Flags:

    [ ] 0x00000001 Special pool.
    [ ] 0x00000002 Force IRQL checking.
    [ ] 0x00000004 Randomized low resources simulation.
    [ ] 0x00000008 Pool tracking.
    [ ] 0x00000010 I/O verification.
    [ ] 0x00000020 Deadlock detection.
    [ ] 0x00000080 DMA checking.
    [ ] 0x00000200 Force pending I/O requests.
    [ ] 0x00000400 IRP logging.
    [ ] 0x00000800 Miscellaneous checks.

The new settings are in effect until the system
is restarted or the settings are changed again.

```